From c862ec13473625a37edb761c7bc5c62f0917f067 Mon Sep 17 00:00:00 2001
From: test <test@test.com>
Date: Thu, 13 Aug 2026 15:56:28 +0200
Subject: [PATCH] fix a typo

---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index 0a63820..204b7f9 100644
--- a/README.md
+++ b/README.md
@@ -1 +1 @@
-# This is my awesom project!
+# This is my awesome project!
-- 
2.47.3
